bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Flight/Profile/Model/Transform.cpp

```diff
@@ -127,8 +127,8 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Flight_Profile_Model_Transform(pybin
         )
 
         .def_static(
-            "nadir_pointing",
-            &Transform::NadirPointing,
+            "local_orbital_frame",
```

Comment:
Perhaps rename to `local_orbital_frame_pointing` for consistency?
Also update the docstrings accordingly and apply to the rest of the files